src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,11 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under construction';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
+});
```

Comment:
- [ ] According to the project requirement, it was stated that our categories should Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)

Suggested solution :point_down:
```JS
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer
```

Reply:
Thank you for taking the time to review my code. I really appreciate the valuable insights you provided 🙇 . I'll make sure to incorporate your recommendations promptly.